src/linked_list.py

```diff
@@ -72,21 +72,21 @@ def display(self):
         linked_list_string += ")"
         return linked_list_string
 
-    def remove(self, node):
+    def remove(self, value):
```

Comment:
Unfortunately, the assignment specs say that `remove` should accept a node argument, rather than a value. I think that's pretty silly personally, but that's what the specs say (and also what I test for).

Also, make sure to consider testing for the edge case where someone tries to remove a node which is not in your linkedlist.